README.md

```diff
@@ -2,110 +2,183 @@
 
 [![Build Status](https://travis-ci.org/commercetools/commercetools-sunrise-data.svg?branch=master)](https://travis-ci.org/commercetools/commercetools-sunrise-data)
 
-## How to create the data with the commercetools impex tool
-
-### csv files encoding:
- - delimited by ","
- - encoding / file origin: UTF-8
- - use text data format for ALL columns
- 
-### import product types
-1. Import it with https://impex.sphere.io/playground
- - Endpoint: Product Types
- - Method: Create
- - use product-types/product-types.json as body, **but remove the surrounding square brackets before**
-
-### import categories
-1. Use the category structure as in categories.csv
-
-2. Import it with https://impex.sphere.io/commands/category-import
-
-### import products
-1. Use the product structure as in products.csv
-2. Import it with https://impex.sphere.io/commands/product-import
-
 
+## How to create a project with Sunrise data
+
+Before starting the import, make sure you have access to the [Admin Center](https://admin.commercetools.com) and the [IMPEX tool](https://impex.commercetools.com/). You will also need to run the application in this repository, check that you meet the [requirements](#requirements).
+
+### 1. Set up your project
+1. Open the [Admin Center](https://admin.commercetools.com) and create an empty project (without sample data).
+2. Select your new project and go to **`Settings`**:
+    - In the tab **`International`**:
+        - **Currencies**: Euro (EUR)
+        - **Countries**: Germany (DE), Austria (AT)
+        - **Languages**: German (DE), English (EN)
+        - **Zone**: Europe with DE, AT
+    - In the tab **`Taxes`**:
+        - **Tax Category name**: "standard"
+        - **Tax rates**:
+            - Germany: 19% incl. in price
+            - Austria: 20% incl. in price
+    - In the tab **`Shipping Methods`**:
+        - **Standard shipping**:
+            - Name: "Standard"
+            - Description: "Delivery in 5-6 working days"
+            - Price: 3 EUR
+            - Free above: 200 EUR
+            - Set as default: true
+        - **Express shipping**:
+            - Name: "Express"
+            - Description: "Delivery the same day"
+            - Price: 10 EUR
+    - In the tab **`Customer Groups`**:
+        - **Name**: "b2b"
+        
+### 2. Import channels
```

Comment:
Is the channel import really necessary as a separate step?   I always did it in one run and always create the channels independently of whether I also create inventory and prices for them (which would be nice to not have as a default)

Reply:
Unfortunately with the current import job for products it's mandatory (or the prices would fail for those channels)... And it's annoying because it's the part that mostly requires the import app. I've created a ticket to find solutions https://github.com/commercetools/commercetools-sunrise-data/issues/82